### DISCUSSION TITLE
deploy wasm project on iis ?

### DISCUSSION THREAD

**Discussion by lavahasif** [unoplatform/uno#10626] [Q&A] [answered]
After deploy wasm project on iis showing 
HTTP Error 500.19 - Internal Server Error
The requested page cannot be accessed because the related configuration data for the page is invalid.

**Comment by jeromelaban** [comment, marked as answer]
See here [for additional details](https://platform.uno/docs/articles/how-to-host-a-webassembly-app.html?q=server#iis). Otherwise, you'll need to configure IIS to display more information about the error.

**Reply by lavahasif** [reply]
tHANK IT WORKED.
ANOTHER PROBLEM.


![image](https://user-images.githubusercontent.com/22430922/205299619-fda133e1-4e8c-4921-a624-1948069365e0.png)

```XML    <staticContent>
      <remove fileExtension=".dll" />
      <remove fileExtension=".wasm" />
      <remove fileExtension=".woff" />
      <remove fileExtension=".woff2" />
      <mimeMap fileExtension=".wasm" mimeType="application/wasm" />
      <mimeMap fileExtension=".clr" mimeType="application/octet-stream" />
      <mimeMap fileExtension=".pdb" mimeType="application/octet-stream" />
      <mimeMap fileExtension=".woff" mimeType="application/font-woff" />
      <mimeMap fileExtension=".woff2" mimeType="application/font-woff" />
      <mimeMap fileExtension=".dat" mimeType="application/octet-stream" />
      <!-- Required for PWAs -->
      <!-- <mimeMap fileExtension=".json" mimeType="application/octet-stream" /> -->
    </staticContent>```


THIS LINE MAKE PROBLEM I COMMENT IT .THEN IT WORKED WHAT IS BEHIND.
 <!-- Required for PWAs -->
` <mimeMap fileExtension=".json" mimeType="application/octet-stream" /> `

**Reply by jeromelaban** [reply]
This only means that `.json` is already registerd by your default config, you can remove it.

**Reply by lavahasif** [reply]
> This only means that `.json` is already registerd by your default config, you can remove it.

![image](https://user-images.githubusercontent.com/22430922/205305750-8f2082d6-1e4e-4469-8c16-6f19f53ff02c.png)
yES IT INHERITED.
